src/odemis/gui/cont/tabs/sparc_acquisition_tab.py

```diff
@@ -290,11 +290,13 @@ def __init__(self, name, button, panel, main_frame, main_data):
         # add "Use scan stage" check box if scan_stage is present
         sstage = main_data.scan_stage
         if sstage:
-            # Move the scan stage to the center (so that scan has maximum range)
             ssaxes = sstage.axes
             posc = {"x": sum(ssaxes["x"].range) / 2,
                     "y": sum(ssaxes["y"].range) / 2}
-            sstage.moveAbs(posc)
+            # In case of a 'real' scan stage, move the scan stage
+            # to the center (so that scan has maximum range)
+            if main_data.stage.name not in sstage.affects.value:
+                sstage.moveAbs(posc)
```

Comment:
It seems posc is not used any where else, so you can move it inside the `if` block.

Reply:
Well it is actually used in line 311-314 as well or do I not interpret you comment correctly?

Reply:
Sorry, missed that.